### ISSUE TITLE
FramePool error infinitely ticking, slowing down the game

### ISSUE THREAD

**Issue by Dezaku** [Questie/Questie#5274]
## Bug description

I started the game after running the update all button in WoWUp (might have updated Questie) then started my game. Now I get a error which infinitely ticks and slows down my game

## Screenshots

![image](https://github.com/Questie/Questie/assets/68233398/db33a04d-dad4-4c6a-ac03-bc4b3f3b50c2)

## Error Message
```
Message: ...ddOns/Questie/Modules/FramePool/QuestieFramePool.lua:216: Wrong object type for function
Time: Tue Dec 12 12:42:34 2023
Count: 11422
Stack: ...ddOns/Questie/Modules/FramePool/QuestieFramePool.lua:216: Wrong object type for function
[string "=[C]"]: ?
[string "=[C]"]: in function `CreateFrame'
[string "@Interface/AddOns/Questie/Modules/FramePool/QuestieFramePool.lua"]:216: in function `CreateLine'
[string "@Interface/AddOns/Questie/Modules/FramePool/QuestieFramePool.lua"]:186: in function `CreateWaypoints'
[string "@Interface/AddOns/Questie/Modules/Map/QuestieMap.lua"]:961: in function `DrawWaypoints'
[string "@Interface/AddOns/Questie/Modules/Map/QuestieMap.lua"]:400: in function `ShowNPC'
[string "@Interface/AddOns/Questie/Modules/QuestieMenu.lua"]:116: in function <Interface/AddOns/Questie/Modules/QuestieMenu.lua:109>

Locals: (*temporary) = <function> defined =[C]:-1
```

## Questie & WoW version
 9.2.0
--> SoD

**Comment by Dezaku** [comment]
Sorry, didn't see that it was already reported here on GitHub, closing down this issue!

**Comment by Muehe** [comment]
@Dezaku Disabling vendors from the minimap button should do as a workaround until we have a proper fix.

**Comment by Dezaku** [comment]
> @Dezaku Disabling vendors from the minimap button should do as a workaround until we have a proper fix.

That works, thank you!

**Comment by Muehe** [comment]
@Dezaku Fixed in 9.2.1, should be available on the usual channels.
